config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
```

Comment:
You don't need storage config, because all code must be [Pure functions](https://en.wikipedia.org/wiki/Pure_function)
Every dynamic data you'll use must be store in our data collections, and query from it.